### ISSUE TITLE
code completion after '$'

### ISSUE THREAD

**Issue by atanasj** [MikhailArkhipov/vscode-r#37]
Code completion does not seem to work when using the `$` symbol i.e., `df$` does not offer a list of variables. Is this a known limitation or am I doing something wrong? 

Relevant info: 

```
ProductName:	Mac OS X
ProductVersion:	10.14.6
BuildVersion:	18G95

1.38.0
3db7e09f3b61f915d03bbfa58e258d6eee843f35
x64

R version 3.6.1 (2019-07-05) -- "Action of the Toes"
Copyright (C) 2019 The R Foundation for Statistical Computing
Platform: x86_64-apple-darwin15.6.0 (64-bit)
```

Thanks.

**Comment by MikhailArkhipov** [comment]
It depends. Basic `$` works

![image](https://user-images.githubusercontent.com/12820357/77282506-59ba7c80-6c87-11ea-9c7d-81efe7f7d337.png)

However, there is no code flow analysis so 
```r
x = mtcars
x$
```
won't bring anything

**Comment by MikhailArkhipov** [comment]
There is no code flow analysis at the moment, so completions don't always know what `x` is as it does not track assignments. It is a bit better with RTVS since it sources code to internal session on `ctrl+enter` and then completions pick it up. With sourcing to terminal it does not help. Starting with 0.0.11 you can try and source code to internal session with `ctrl+shift+enter` and see if that helps.

**Comment by MikhailArkhipov** [comment]
Should be fixed in 0.0.15. Source the file, then completions should be working.

![image](https://user-images.githubusercontent.com/12820357/93031083-02b1b680-f5dd-11ea-8391-7bb0a6ff3547.png)
